omnibus/config/templates/updater/datadog-agent-exp.service.erb

```diff
@@ -9,12 +9,13 @@ Wants=datadog-agent-trace-exp.service datadog-agent-process-exp.service datadog-
 
 [Service]
 Type=oneshot
-PIDFile=<%= install_dir %>/run/agent.pid
+PIDFile=<%= agent_dir %>/run/agent.pid
 User=dd-agent
 EnvironmentFile=-<%= etc_dir %>/environment
-ExecStart=<%= install_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= install_dir %>/run/agent.pid
-ExecStart=<%= install_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= install_dir %>/run/agent.pid
-ExecStart=<%= install_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= install_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/bin/agent/agent run -p <%= agent_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/bin/agent/agent run -p <%= agent_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/bin/agent/agent run -p <%= agent_dir %>/run/agent.pid
+ExecStart=<%= agent_dir %>/agent_entrypoints/experiment_agent/agent run -p <%= agent_dir %>/run/agent.pid
```

Comment:
Why is `ExecStart` provided 4 times?

Reply:
It's provided multiple times as the main experiment file is a oneshot.
It launches the core-agent and if the core-agent returns with a 0 exit code, then the other ExecStart are run.
This results in 3 runs of the core-agent, and the forth ExecStart forces a failure that triggers back the stable datadog-agent